Source/Core/DolphinWX/Input/InputConfigDiag.h

```diff
@@ -105,13 +105,13 @@ class ControlDialog : public wxDialog
 {
 public:
   ControlDialog(InputConfigDialog* const parent, InputConfig& config,
-                ControllerInterface::ControlReference* const ref);
+                ControlReference* const ref);
 
   bool Validate() override;
 
   int GetRangeSliderValue() const;
 
-  ControllerInterface::ControlReference* const control_reference;
+  ControlReference* const control_reference;
```

Comment:
I don't mind if you do, but they should be cleaned up at some point - and I prefer doing that sooner than later, especially when you're touching them anyways.
But I also understand that you're not that familiar with it yet and want to do it later once you've gained more understanding of it.